client/account-recovery/lost-password/index.jsx

```diff
@@ -0,0 +1,106 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { findDOMNode } from 'react-dom';
+import { localize } from 'i18n-calypso';
+import classnames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import support from 'lib/url/support';
+import Card from 'components/card';
+import Button from 'components/button';
+import FormLabel from 'components/forms/form-label';
+import FormInput from 'components/forms/form-text-input';
+
+export class LostPassword extends Component {
+	constructor() {
+		super( ...arguments );
+
+		this.state = {
+			isSubmitting: false,
+		};
+	}
+
+	onSubmit = () => {
+		const userLogin = findDOMNode( this.refs.user_login ).value;
```

Comment:
Based on the component's usage, I'd probably recommend making it a controlled input, setting value into state `onChange` and submitting/rendering using the state value.

Else you could have a parent component manage the value in its state (using on `onChange` prop here), or manage via Redux state (probably overkill given the amount of boilerplate involved).